packages/badge/Badge.test.tsx

```diff
@@ -1,18 +1,76 @@
-import { Badge } from './Badge';
-
+import { Badge } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
 
 describe( Badge.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Badge!');
-
     return mount(
       <Badge/>
-    ).wait();
+    ).wait((wrapper) => {
+      expect(wrapper.has('.c-badge')).to.equal(true);
```

Comment:
sorry for one additional but let's use chai bdd style guide here http://chaijs.com/api/bdd/#method_true which means `.to.be.true`